### ISSUE TITLE
Convert or move a task to a sub-task

### ISSUE THREAD

**Issue by giammi56** [super-productivity/super-productivity#992]
Pretty much as the title says would be great to have this feature. It would be the natural extension of the already present options "move to to other Project! or "Convert to a parent task", but from a task to a subtask.
It is a crucial feature to reorganize/rearrange tasks over time without losing the time accumulated.
BONUS: DRAG AND DROP to sub-tasks :)

**Comment by johannesjo** [comment]
Thanks for opening this up. Potential duplicate of #905

**Comment by cokxivawo** [comment]
+1 :)

**Comment by nicola-lunghi** [comment]
Here!! is still relevant!!

**Comment by michaelmworthington** [comment]
still relevant. 

agree with potential duplicates, and good discussions on those other issues mentioned.

**Comment by RokeJulianLockhart** [comment]
> https://github.com/johannesjo/super-productivity/issues/992#issuecomment-1876199146

@GitHub-Actions, this remains relevant. Please reopen the issue.

**Comment by RokeJulianLockhart** [comment]
> https://github.com/johannesjo/super-productivity/issues/992#issuecomment-2119058832

@GitHub-Actions, https://github.com/johannesjo/super-productivity/issues/992#issuecomment-1898530602

**Comment by RokeJulianLockhart** [comment]
> https://github.com/johannesjo/super-productivity/issues/992#issuecomment-2357335267

It remains relevant.

**Comment by RokeJulianLockhart** [comment]
> https://github.com/johannesjo/super-productivity/issues/992#issuecomment-2600470515

See https://github.com/johannesjo/super-productivity/issues/992#issuecomment-2357956434.

**Comment by beedell-roke** [comment]
> https://github.com/johannesjo/super-productivity/issues/992#issuecomment-3095041271

See https://github.com/johannesjo/super-productivity/issues/992#issuecomment-2357956434.

**Comment by Malanius** [comment]
Was just shuffling some things around, one of my side-projects got big enough to receive full full-fledged project. Up until this point, it was just a task with subtasks, but it grew bigger than expected. So I wanted to move all the subtasks to new parent tasks (like epics for project modules) and found out this is not possible.

It's easy to recreate ones not yet started in proper places, but ones that have already been started and have some time tracked and notes on them are not as straightforward and having this option would help a lot.

**Comment by muaddib32** [comment]
Yes, this would be great. As a new user I expected I would be able to do this and was surprised when I couldn't.

**Comment by SqrtNegativOne** [comment]
I would like this too. Had to re-make a lot of tasks. Once again, my heartfelt thanks to the developer!

**Comment by reedstrm** [comment]
New user here. Was specifically looking for/hoping for help with task breakdown. Seems superproductivity is having issues growing past its time-tracker roots. Looks really nice for that, so far, but promoting and demoting tasks is core to how I organize.

**Comment by MooMarMouse** [comment]
I would also like this please :)

**Comment by TJMcK** [comment]
Hoping to see this feature in the future (within a year if possible)

**Comment by ReFil** [comment]
Related to this, you can drag and drop items into subtasks _but only if the parent task already has a subtask_. If the parent task has no subtasks you cannot drag a task into it. You must create a subtask first. This is not how one would expect it to work, ideally it should permit creating a subtask

**Comment by johannesjo** [comment]
This has since been implemented.

**Comment by beedell-roke** [comment]
@ReFil, that may be worth a separate issue, if it persists.
